arenadata/readme.md

```diff
@@ -5,36 +5,25 @@
 Change directory to gpdb sources destination. Make sure that directry doesn't contain binary objects from previous builds. Then run:
 for Ubuntu:
 ```bash
-docker build -t gpdb7_u22:latest -f arenadata/Dockerfile.ubuntu .
-```
-for Rocky Linux:
-```bash
-docker build -t gpdb7_regress:latest -f arenadata/Dockerfile .
+docker build -t gpdb8_u22:latest -f arenadata/Dockerfile.ubuntu .
 ```
 
 CI pushes docker images to the internal registry for each branch. We can pull it with usage of:
 
-* branch name as tag (latest for `adb-7.x` branch)
+* branch name as tag (latest for `adb-8.x` branch)
 * commit hash:
   ```bash
-  docker pull hub.adsw.io/library/gpdb7_regress:1353d81
+  docker pull hub.adsw.io/library/gpdb8_regress:1353d81
```

Comment:
may be `:latest`?

Reply:
This is just an example. The same hash is specified for 6X and 7X.

Reply:
Yes, but why set a bad example?

Reply:
Here is an example of downloading an image using the commit hash, so the latest will be incorrect. I can replace it with `abc1234` to make it more obvious that this is not a real hash.

Reply:
Okay, I understand you.